subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -621,16 +621,32 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_pdu *rx)
 	 * with a valid context.
 	 */
 	if (ftr->aux_lll_sched) {
-		/* AUX_ADV_IND/AUX_CHAIN_IND PDU reception is being setup */
-		ftr->aux_sched = 1U;
-
 		if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_PERIODIC) && sync_lll) {
 			sync_lll->lll_aux = lll_aux;
 
+			/* AUX_ADV_IND/AUX_CHAIN_IND PDU reception is being
+			 * setup
+			 */
```

Comment:
What is the reasoning behind not configuring your IDE / not using the 100 column width? 

As per the coding guidelines this isn't a blocking change, but just curious :)

Reply:
Those are fair points for why we should have stayed at 80 instead of 100 for Zephyr, but are not very good points for not following our coding guidelines :) 

Either way, it doesn't really hurt